src/components/PageHelpFeedback.tsx

```diff
@@ -0,0 +1,206 @@
+
+export type FeedbackState = 'initial' | 'yes' | 'no' | 'submitted'
+
+export default function PageHelpFeedback({ className }: { className?: string }) {
+  const [slug, setSlug] = useState('')
+  React.useEffect(() => {
+    if (typeof window !== 'undefined') {
+      setSlug(window.location.pathname)
+    }
+  }, [])
+  const [state, setState] = useState<FeedbackState>('initial')
+  const [yesText, setYesText] = useState('')
+  const [noOption, setNoOption] = useState('')
+  const [noText, setNoText] = useState('')
+  const [submitting, setSubmitting] = useState(false)
+  const [lastFeedbackWasYes, setLastFeedbackWasYes] = useState(false)
+
+  const handleSubmit = async () => {
+    setSubmitting(true)
+    const basePath = process.env.NEXT_PUBLIC_BASE_PATH || ''
+    await fetch(`${basePath}/api/feedback`, {
+      method: 'POST',
+      headers: { 'Content-Type': 'application/json' },
+      body: JSON.stringify({
+        slug,
+        helpful: state === 'yes',
+        yesText: state === 'yes' ? yesText : undefined,
+        noOption: state === 'no' ? noOption : undefined,
+        noText: state === 'no' ? noText : undefined,
+      }),
+    })
+    setSubmitting(false)
+    setState('submitted')
+    if (state === 'yes') {
+      setLastFeedbackWasYes(true)
+      setTimeout(() => setLastFeedbackWasYes(false), 1000)
+    }
+    setYesText('')
+    setNoOption('')
+    setNoText('')
+  }
+
+  return (
+    <>
+      <div
+        className={`flex flex-col items-start gap-2 text-sm w-full ${className ?? ''}`}
+        style={{ minHeight: 90 }}
+      >
+        <div className="flex flex-row items-center gap-x-3 w-full">
+          <h3 className="font-medium text-gray-800 m-0 flex items-center">
+            Was this page helpful?
+          </h3>
+          <div className="flex gap-0">
+            <Button
+              aria-label="Yes"
+              variant={state === 'yes' ? 'secondary' : 'tertiary'}
+              size="small"
+              className={`rounded-lg hover:bg-grayAlpha-200 transition-colors ${state === 'yes' ? 'bg-gray-200 border border-gray-400' : ''} ${state === 'submitted' && lastFeedbackWasYes ? 'wiggle' : ''}`}
+              onClick={() => setState(state === 'yes' ? 'initial' : 'yes')}
+              disabled={submitting}
+            >
+              <ThumbsUp className="w-4 h-4" />
+            </Button>
+            <Button
+              aria-label="No"
+              variant={state === 'no' ? 'secondary' : 'tertiary'}
+              size="small"
+              className={`rounded-lg hover:bg-grayAlpha-200 transition-colors ${state === 'no' ? 'bg-gray-200 border border-gray-400' : ''}`}
+              onClick={() => setState(state === 'no' ? 'initial' : 'no')}
+              disabled={submitting}
+            >
+              <ThumbsDown className="w-4 h-4" />
+            </Button>
+          </div>
+        </div>
+
+        <style jsx global>{`
```

Comment:
Don't know if this is a good way to do it. Maybe we can make this a Tailwind animation we can reuse?